src/_sass/tilt2.scss

```diff
@@ -3,35 +3,44 @@
 $tilt2-header-width: 1080px;
 $tilt2-page-width: 1080px;
 
+$tilt2-color-off-white: #FAFBFC;
 $tilt2-color-gray-lightest: #F1F1F1;
 $tilt2-color-gray-lighter: #B4B4B4;
 $tilt2-color-gray: #586E75;
 $tilt2-color-gray-dark: #073642;
 
+$tilt2-alpha-transparent: 0.3;
+
 $tilt2-color-text: $tilt2-color-gray-dark;
 
 
 // TYPOGRAPHY
 // --------------------------------------------------
-$tilt2-font-sans: 'Noto Sans', noto-sans, sans-serif;
+$tilt2-font-sans: noto-sans, sans-serif;
+$tilt2-font-serif: merriweather, serif;
 $tilt2-font-mono: 'Fira Code', monospace;
 
+$tilt2-font-weight-sans-light: 300;
+$tilt2-font-weight-sans-bold: 700;
+$tilt2-font-weight-sans-black: 900;
```

Comment:
I'm generally skeptical of adding a new font weight that we only use in one place. It seems messy from both a design perspective (because it, by definition, breaks out of the existing design system) and an eng perspective (because we have to download a new font for each new weight).

Reply:
I'm not 100% sure I'm interpreting this comment correctly, but I think you're saying that we should have fewer font weights. I tried to make the design work with just two sans-serif font weights — light and bold. But I think the black really makes the hero heading a lot more impactful. It's the first thing you see, and the effect is quite different (and worse, I think) in bold.

I'm re-reading this comment, though, and I could interpret it to mean that you want the font-weights to be more closely associated with each type of font? i.e., I only want 300, 700, and 900 to be available for use with our sans font "Noto Sans" (and not our mono or serif font). If that's the case, I would actually like to make that usage more clear in the shared SCSS but I haven't yet gotten around to it...

Reply:
Oh, I've also seen a neat trick where you can download only the few characters you need from a Google Font. Like so:

~~~
http://fonts.googleapis.com/css?family=Inconsolata&text=MyHeader
~~~

But the Google Fonts version of Noto Sans lacks the "black" weight we want. So I'm using the Adobe Typekit version of Noto Sans. And the `&text` trick doesn't seem to work there.

Reply:
more the first one. I think the way we've addressed this in the past is to use a darker color rather than a bolder font

i don't feel strongly about it though, it's your decision

Reply:
(tho your second reading is also a good point that I wish I had said! We should probably have font mixins that group family + weight together; you almost never want to set them independently)